src/workerd/api/actor.c++

```diff
@@ -203,4 +203,36 @@ kj::Own<IoChannelFactory::ActorClassChannel> DurableObjectClass::getChannel(IoCo
   KJ_UNREACHABLE;
 }
 
+void DurableObjectClass::serialize(jsg::Lock& js, jsg::Serializer& serializer) {
+  auto& handler = JSG_REQUIRE_NONNULL(serializer.getExternalHandler(), DOMDataCloneError,
+      "DurableObjectClass cannot be serialized in this context.");
+  auto externalHandler = dynamic_cast<Frankenvalue::CapTableBuilder*>(&handler);
+  JSG_REQUIRE(externalHandler != nullptr, DOMDataCloneError,
+      "DurableObjectClass cannot be serialized in this context.");
```

Comment:
Nit: this pattern (getting the external handler, checking that it's not null, and throwing the `DOMException` is common enough and repeated enough now that a utility macro or function to eliminate the duplication would be nice.

Reply:
I agree but I think I'm going to punt. I actually want to go back and rename KJ's `dynamicDowncastIfAvailable` to `dynamicDowncast` (since we always enable RTTI these days), and then use it instead of `dynamic_cast` in all these places. And then yeah stacking some helpers on top of that for checking serialization probably makes sense.